### DISCUSSION TITLE
Cannot change the location on raspi

### DISCUSSION THREAD

**Discussion by dilsg** [aaronwmorris/indi-allsky#902] [Q&A] [answered]
Hi. I have changed the postion of my raspberry cam to 51.1 and -13.6. But allways it is set back to 51 and -2.3.
I have restarted the setup.sh to change the GPS-Part in indiserver from GPF Off to simulated GPS. But no better result.
Is the a way to set my values in a conf- or json-file?
Best regards from Dresden
Gunter

**Comment by aaronwmorris** [comment, marked as answer]
You have the GPS simulator enabled.  Re-run `setup.sh` and select "None" for GPS.  You will then need to manually restart the indiserver or just reboot.

```
systemctl --user restart indiserver
```

**Comment by dilsg** [comment]
Thank you for your answer. I tried it, but the system doesn't change the behavior. I guess it will not change this item in the database.
Can I find the values in the database? I would look at them.

**Reply by aaronwmorris** [reply]
The location you see in the web information panel (lower left side) is the location associated with the selected camera.  The location in the config is synchronized to the camera when the indi-allsky service is started or reloaded.

The config has to be "reloaded" or service restarted to have the updated latitude and longitude reflected in the web interface.

You can view the camera latitude and longitude using the following commands:

```
sqlite3 /var/lib/indi-allsky/indi-allsky.sqlite "SELECT id,name,latitude,longitude FROM camera;"
```

**Comment by dilsg** [comment]
Hi - and thank you for your comment. I solved it now with the SQL-Statement: 
`update camera set longitude=-13.6 where id=1`

Now it works great.

**Reply by dilsg** [reply]
Oh - after reboot, there is the old value in logitude. 
What a pitty! 
I guess the system - raspi4, with HQ-Cam - is thinking, there is a GPS-sensor inside and sets the location.

**Comment by aaronwmorris** [comment]
On the CLI can you run the following and show me the results?

```
ps auxww | grep indiserver
```

**Comment by dilsg** [comment]
Hi Aaron -
thank you for your great help. Now I found my mistake.
I forgot to stop the indiserver and start then the setup.sh. 
Now it is all fine. 
Thank you -
here now my result:

pi@raspi188:~ $ ps auxww | grep indiserver
pi           776  0.0  0.0   9236  3268 ?        Ss   19:45   0:00 /usr/local/bin/indiserver -p 7624 indi_simulator_telescope indi_simulator_ccd
pi          2291  0.0  0.0   6064   648 pts/0    S+   19:50   0:00 grep --color=auto indiserver


Greetings from Dresden - it is a beautiful city
